scripts/upgradeDependencies/parseUpgradedPackages.js

```diff
@@ -1 +1,35 @@
-module.exports = output => [...new Set(output.match(/^.+?(?=:\s{$)/gm))];
+const outputReducer = (accumulator, curr) => {
```

Comment:
Some comments in this file would go a long way in helping others understand what these functions are doing.